app/views/comments/_comments.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3>Comments</h3>
+<% if can? :create, comment %>
+<div id="comment_form" class="mb-4">
+  <%= render partial: 'comments/comment_form', locals: { resource: resource, comment: comment } %>
+</div>
+<% end %>
+<p><%= pluralize(comments.length, 'Comment') %></p>
+<% comments.each do |icomment| %>
+<div id="comment_<%= icomment.id %>" class="card mb-2">
+<%= render partial: 'comments/comment', locals: { comment: icomment } %>
+</div>
+<% end %>
+<%== pagy_bootstrap_nav(pagy) %>
```

Comment:
<%== what is this extra = for?

Reply:
It is like equivalent of raw or html_safe